dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -172,8 +172,8 @@ func main() {
 		os.Exit(0)
 	}
 
-	if len(*dashboard) > 0 {
-		err := Export(client, *kibanaURL, *dashboard, *fileOutput)
+	if len(*id) > 0 {
+		err := Export(client, *kibanaURL, *id, *fileOutput)
 		if err != nil {
 			fmt.Printf("ERROR: fail to export the dashboards: %s\n", err)
```

Comment:
Can you double check all the exit codes used in this file to ensure that when there is an error that the value is non-zero.

And check that os.Stderr is used to log errors.

Reply:
Did a quick "cleanup" I think we have to do a full cleanup to this file at one stage.

Is `os.Stderr` because of `mage`?

Reply:
Thanks. It will help integrating it with other tools if it follows the convention of writing errors to [stderr](https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)).

And yes, if your follow these conventions if will integrate nicely with `mage` which by default passes through the stderr output. Stdout is only passed through when executing in verbose mode (`mage -v`).